nf_core/__main__.py

```diff
@@ -201,7 +201,7 @@ def launch(pipeline, id, revision, command_only, params_in, params_out, save_all
 
 # nf-core download
 @nf_core_cli.command(help_priority=3)
-@click.argument("pipeline", required=True, metavar="<pipeline name>")
+@click.argument("pipeline", required=False, metavar="<pipeline name>")
```

Comment:
`False` is the default, so we can just omit this.

```suggestion
@click.argument("pipeline", metavar="<pipeline name>")
```